app/helpers/content_header_helper.rb

```diff
@@ -0,0 +1,211 @@
+module ContentHeaderHelper
+  class AbstractContent
+    include ActionView::Helpers
+    attr_accessor :output_buffer
+
+    delegate :key_for, to: OpenProject::AccessKeys
+  end
+
+  class ContentHeader < AbstractContent
+    def initialize(title:, subtitle: '', helper: nil, scrollable: false, &_block)
+      @title = title
+      @subtitle = subtitle
+      @helper = helper
+      @scrollable = scrollable
+    end
+
+    def toolbar(&block)
+      content_tag :div, class: 'toolbar-container' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.